routing/nexus_config.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Orgchart{
```

Comment:
Unsure if namespace and class name can cause collision with LEAF_Nexus/config.php

Reply:
We wont be using that file going forward.